packages/frontend/core/src/modules/cloud/services/auth.ts

```diff
@@ -116,13 +116,18 @@ export class AuthService extends Service {
   ) {
     track.$.$.auth.signIn({ method: 'magic-link' });
     try {
+      const scheme = BUILD_CONFIG.isElectron
+        ? appInfo?.schema
+        : BUILD_CONFIG.isIOS || BUILD_CONFIG.isAndroid
+          ? 'affine'
+          : 'web';
```

Comment:
Are we planning to submit different app versions for canary/beta/stable?

Reply:
Scheme in electron is dynamic for each channel, but it's fixed for iOS. Not sure about android yet.